src/libmain.c

```diff
@@ -140,7 +140,7 @@ static GOptionEntry entries[] =
 	{ "no-session", 's', G_OPTION_FLAG_REVERSE, G_OPTION_ARG_NONE, &cl_options.load_session, N_("Don't load the previous session's files"), NULL },
 #ifdef HAVE_VTE
 	{ "no-terminal", 't', 0, G_OPTION_ARG_NONE, &no_vte, N_("Don't load terminal support"), NULL },
-	{ "vte-lib", 0, 0, G_OPTION_ARG_FILENAME, &lib_vte, N_("Filename of libvte.so"), NULL },
+	{ "vte-lib", 0, 0, G_OPTION_ARG_FILENAME, &lib_vte, N_("Use `FILE' as the dynamically-linked VTE library"), N_("FILE") },
```

Comment:
I don't quite like <code>&#96;'</code> quotes, can't we stick to `''` or `""`? (or here I don't even really feel the need for quotes at all, yet I guess it does make sense to have them)

Reply:
Done.